model.py

```diff
@@ -31,16 +31,26 @@ def device(self):
         return next(self.parameters()).device
 
     @argbind.bind
-    def sample(self, batch_size=1, num_steps=100, bias=0.0):
+    def sample(self, batch_size=1, num_steps=100, bias=0.0, stddev=0.1):
         x = torch.zeros(1, batch_size, 3).to(self.device)
         hidden = None
         output = []
         for i in range(num_steps):
             x, hidden = self.step(x, hidden)
             output.append(x)
-        return torch.vstack(output)
-
-
+        output = torch.vstack(output)
+        zeros = torch.zeros(num_steps, batch_size, 2).to(self.device)
+
+        # Bias is weird now because we have a fixed stddev.  Instead of using bias, we could
+        # just adjust stddev.
+        #
+        # However, in the future, log-stddev will be part of the output of the model, and so it makes sense
+        # to apply bias independently.
+        std = torch.exp(torch.log(zeros + stddev) - bias)
```

Comment:
The following lines need to happen inside the for loop.

Reply:
Fixed